device/core/devdoc/device_client_requirements.md

```diff
@@ -250,6 +248,8 @@ interface DeviceMethodEventHandler {
 
 **SRS_NODE_DEVICE_CLIENT_16_086: [** Any operation (such as `sendEvent` or `onDeviceMethod`) happening after a `setRetryPolicy` call should use the policy set during that call. **]**
 
+**SRS_NODE_DEVICE_CLIENT_16_096: [** The `setRetryPolicy` method shall call the `setRetryPolicy` method on the twin if it is set and pass it the `policy` object. **]**
```

Comment:
>setRetryPolicy [](start = 81, length = 14)

We have a requirement that says that calling getTwin after setRetryPolicy will use that retry policy on the newly created twin, along with tests.  (maybe we already do....)

Reply:
we do in the twin requirements.

---
In reply to: [171947064](https://github.com/pierreca/azure-iot-sdk-node/pull/2#discussion_r171947064) [](ancestors = 171947064)